datasource/remote/src/main/java/com/egdroid/datasource/remote/movie/PopularMovieService.kt

```diff
@@ -0,0 +1,12 @@
+package com.egdroid.datasource.remote.movie
+
+import com.egdroid.models.remote.movie.popularmovie.MovieResponse
+import retrofit2.Call
+import retrofit2.http.GET
+
+
+interface PopularMovieService {
+
+    @GET("popular?api_key=828157095a54ed8f68b8882624ed7660&language=en-US&page=1")
```

Comment:
We should not leave private keys to public access. Let's add it to any ignored file (i.e. `local.properties`)

Reply:
Let's use a retrofit interceptor like @Ahmed-Abdelmeged mentioned in the session.